src/components/animated/_spotlight.less

```diff
@@ -0,0 +1,201 @@
+.sbtn.spotlight-btn{
+    /* @mrsidsat */
+    position: relative;
+    padding:0px;
+    color:black;
+    border:3px solid #0cbfcf ;
+    height:50px;
+    width:200px;
+    background-color: transparent;
+    overflow:visible !important;
+    cursor:pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-shadow: inset 0px 0px 10px  #0fe0f5;  
+	.blue-btn{
+		border:3px solid blue;
+		box-shadow: inset 0px 0px 10px blue; 
```

Comment:
The code inside each of the button is the same but with different color so instead of repeating the code, create a mixin that takes in the color as a parameter and that has this block of code, then call it inside each of the color button with the appropriate color.

Also instead of this line:
```
border:3px solid blue;
```
use this:
```
border-color: blue;
```

And instead of using literal names like "blue" you need to use the color variables that we have in this project like `@blue` and so on.